src/main/java/io/lettuce/core/tracing/NoOpTracing.java

```diff
@@ -55,11 +60,30 @@ public boolean isEnabled() {
         return false;
     }
 
+    @Override
+    public TracingTagsCustomizer getTracingTagsCustomizer() {
+        return NOOP_TRACING_TAGS_CUSTOMIZER;
+    }
+
     @Override
     public Endpoint createEndpoint(SocketAddress socketAddress) {
         return NOOP_ENDPOINT;
     }
 
+    static class NoOpTracingTagsCustomizer implements TracingTagsCustomizer {
```

Comment:
I guess this class could be made public in case folks want to do tracing but dont care about tags.